### ISSUE TITLE
Admin ability to sort by open requests vs. closed requests.

### ISSUE THREAD

**Issue by blipblitz** [MuckRock/muckrock#1425]
Would like to filter by all AA/AR/FR and by all Completed/Rejected/NRD.

**Comment by morisy** [comment]
This feature would be broadly useful beyond just admins. Not sure best way to implement it without making that area even more complex, but worth doing one way or another.

**Comment by mitchelljkotler** [comment]
dupe #1385

**Comment by mitchelljkotler** [comment]
I agree this is generally useful.  I started looking into this at one point, but the UI always looked bad

1. I think it should be a general filter on statuses, ie you can filter for any subset of statuses you want, not just predefined groups of open or closed
2. Ideas I had to implement this is listing all statuses with checkboxes (simple, but very verbose) or using some sort of autocomplete, which isn't great due to you not being able to see the choices initially... didn't want to introduce a new widget just for this.  Maybe incorporate into a larger front end redesign.
